experiments/example/main.go

```diff
@@ -58,12 +59,12 @@ func main() {
 
 	// Connect to metadata database (Cassandra is the only supported database).
 	// Besides experiment results and platform metrics (we use Snap to gather them) we save certain deta about experiment configuration and environment (metadata).
-	metadata, err := experiment.NewMetadata(uid, experiment.DefaultMetadataConfig())
+	metaData, err := metadata.NewCassandra(uid, metadata.DefaultCassandraConfig())
 	// errutil.CheckWithContext() is a helper function that will panic on error and provide some additional information about error origin.
 	errutil.CheckWithContext(err, "Cannot connect to Cassandra Metadata Database")
 
 	// Save experiment runtime environment (configuration, environmental variables, etc).
-	err = metadata.RecordRuntimeEnv(experimentStart)
+	err = metadata.RecordRuntimeEnv(metaData, experimentStart)
```

Comment:
It's helper function which uses metadata interface rather than method in this interface.